### DISCUSSION TITLE
Cannot start Cypress

### DISCUSSION THREAD

**Discussion by schwigri** [cypress-io/cypress#18149] [Questions and Help]
Hi, this is my first time using Cypress, but I actually have not been able to use it. I've followed instructions on Cypress website and have installed all dependencies, also I have rebooted my Ubuntu device. However, I receive the following output with strange characters, this is `npx cypress verify`:

    Cypress failed to start.

    This may be due to a missing library or dependency. https://on.cypress.io/required-dependencies

    Please refer to the error below for more details.

    ----------

    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 1: cannot create �T�@�@8
                                                                                @&%@@@����|D�|D��D��T��T��!F�!F�f��������M�M j � ����f�������8@��<�<R�td�f��������  M�      MP�td�լ�լ�լ�8�8Q�td���dd/lib64/ld-linux-x86-64.so.2GNU INFOCrashpad<}GNUגY�o�
    �ڢ�ޓ��M�S�Ba p � ��6d����6���������: Directory nonexistent
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 3: ,: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 4: D: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 5: Z: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 6: p: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 7: z: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 8: �: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 9: �: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 10: �: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 11: �: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 12: �: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 13: �: not found
    /home/griffen/.cache/Cypress/8.4.1/Cypress/Cypress: 14: Syntax error: ")" unexpected

    ----------

    Platform: linux (Ubuntu - 21.04)
    Cypress Version: 8.4.1

If someone can help me troubleshoot and get Cypress working, I would be very grateful!

**Comment by marctaylor700** [comment]
Have you attempted an `npm i`?

**Comment by san-slysz** [comment]
If I were you
- Suppress the Cypress folder from /home/griffen/.cache/ if it exists
- Remove cypress from your package.json, `npm i` (to check your package.json is ok mainly)
- Delete your node_modules folder
- `npm i cypress --save-dev`

That's a way to start clean.
